dubbo-rpc/dubbo-rpc-api/src/main/java/org/apache/dubbo/rpc/RpcStatus.java

```diff
@@ -53,6 +53,12 @@
     private volatile Semaphore executesLimit;
     private volatile int executesPermits;
 
+    /**
+     * Semaphore used to control concurrency limit set by `actives`
+     */
+    private volatile Semaphore activesLimit;

```

Comment:
Is directly init better?
How do u think about it?

Reply:
actives may be changed by URL

Reply:
> Is directly init better?
> How do u think about it?



> Is directly init better?
> How do u think about it?

actives may be changed by url

Reply:
It may be changed by url

Reply:
Ok, u are right.
Pls deal with your code format.

Reply:
ok